orchestrator/api/api_v1/endpoints/processes.py

```diff
@@ -200,8 +206,8 @@ async def resume_all_processess_endpoint(request: Request, user: str = Depends(r
             )
         )
         .filter(ProcessTable.is_task.is_(True))
-        .all()
     )
+    processes_to_resume = list(db.session.scalars(stmt))
```

Comment:
Apparently they moved the `.all()` method into the `ScalarResult` object. Or does it not work the same?

```suggestion
    processes_to_resume = db.session.scalars(stmt).all()
```

Reply:
Ah right, that gives the Sequence result..

Reply:
Looks like they're using [`typing.Sequence`](https://github.com/sqlalchemy/sqlalchemy/blob/2c8a05244a8f7919c3954e0b6e1efaa996792772/lib/sqlalchemy/engine/result.py#L29) though, so should be fine for mypy?

Reply:
Yeah. It's `_async_resume_processes` that wants a list, which is why I just converted. A better way is to to change `_async_resume_processes` (and similar functions) to accept `Sequence` if that is sufficient. `Sequence` is more generic (accepts lists, strings and tuples) and does not support mutations in its interface.

Shall I update function signatures?

Reply:
Yeah, that makes sense.

Interestingly, SqlAlchemy's `ScalarResult.all()` has return type `Sequence` but it calls `ResultInternal._allrows()` which has return type `List` (and it does always return a list). I'm not sure why they would make the return type _less_ specific 🤔